lib/invoice.rb

```diff
@@ -0,0 +1,62 @@
+class Invoice
+  def initialize(buyer:, basket:)
+    @buyer = buyer
+    @basket = basket
+  end
+
+  def get_invoice
+    invoice = ""
+
+    invoice += get_header
+
+    @basket.items.each_with_index do |item, index|
+      invoice += get_basket_item(item, index)
+    end
+
+    invoice += get_summary
+
+    invoice
```

Comment:
name can be better then `get_invoice`, it suggest you download it from somewhere, or fetch from db or disk, but you generate it - `generate_invoice`, `prepare_invoice`, `render_invoice` maybe will be better - please rethink it